compiler/global.cpp

```diff
@@ -451,7 +451,9 @@ void global::reset()
     gGroupTaskSwitch = false;
     gFunTaskSwitch   = false;
 
-    gUIMacroSwitch = false;
+    gUIMacroSwitch    = false;
+    gRustNoTaitSwitch = false;
```

Comment:
Nitpick: `gRustNoTaitSwitch` -> `gRustNoTraitSwitch`?

Reply:
thanks, fixed but this commit got removed from this pr.